packages/office-ui-fabric-react/src/components/Button/CompoundButton/CompoundButton.styles.ts

```diff
@@ -34,12 +34,25 @@ export const getStyles = memoizeFunction((
     },
 
     flexContainer: {
-      flexDirection: 'column',
+      flexDirection: 'row',
       alignItems: 'flex-start',
       minWidth: '100%',
       margin: ''
     },
 
+    textContainer: {
+      textAlign: 'left'
```

Comment:
awesome, this fixes a bug I noticed when compound buttons have multi line descriptions.